ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -55,19 +56,23 @@
   private final StatementParser statementParser;
   private final Duration disconnectCheckInterval;
   private final ObjectMapper objectMapper;
+  private final ActivenessRegistrar activenessRegistrar;
 
   public StreamedQueryResource(
       final KsqlConfig ksqlConfig,
       final KsqlEngine ksqlEngine,
       final StatementParser statementParser,
-      final Duration disconnectCheckInterval
+      final Duration disconnectCheckInterval,
+      final ActivenessRegistrar activenessRegistrar
   ) {
     this.ksqlConfig = ksqlConfig;
     this.ksqlEngine = ksqlEngine;
     this.statementParser = statementParser;
     this.disconnectCheckInterval =
         Objects.requireNonNull(disconnectCheckInterval, "disconnectCheckInterval");
     this.objectMapper = JsonMapper.INSTANCE.mapper;
+    Objects.requireNonNull(activenessRegistrar);
```

Comment:
nit: is good to follow the pattern of providing the param name as the second param to `requireNonNull`. It makes it really clear which param is null - this can be invaluable in support situations.

Search for other places this name needs to be added in your PR too.